riscv/src/compiler.rs

```diff
@@ -552,10 +552,6 @@ fn runtime() -> &'static str {
     }
     */
     r#"
-.globl rust_begin_unwind
-rust_begin_unwind:
-    unimp
-
 .globl memset@plt
 memset@plt:
```

Comment:
It would be nice if you could also add memset/memcpy/memcmp. I'm not sure there is a "proper" way to do this, but we could come up with a unique name for the functions (that does not clash) and then keep this label here and just have `tail <unique name>` here.